trunk/src/app/srs_app_config.cpp

```diff
@@ -6754,6 +6754,20 @@ unsigned short SrsConfig::get_srt_listen_port()
     return (unsigned short)atoi(conf->arg0().c_str());
 }
 
+bool SrsConfig::get_srt_mix_correct() {
+    static bool DEFAULT = true;
+    SrsConfDirective* conf = root->get("srt_server");
+    if (!conf) {
+        return DEFAULT;
+    }
+    
+    conf = conf->get("mix_correct");
+    if (!conf || conf->arg0().empty()) {
+        return DEFAULT;
+    }
+    return SRS_CONF_PERFER_FALSE(conf->arg0());
```

Comment:
应该perfer true，和default保持一致。
```suggestion
    return SRS_CONF_PERFER_TRUE(conf->arg0());
```